readme.md

```diff
@@ -1,55 +1,38 @@
 # K-Link
 
-> the encouraging knowledge management solution
+Connects various information sources to a common digital library and provides search, retrieval and exchange of information from different platforms.
 
-![](./klink-explore-static.png)
+![K-Link Logo](./docs/files/k-link-logo.png)
 
-The [K-Link](https://k-link.technology) is a technology for public organizations, private companies and diverse communities to build strong institutional memories and foster cooperation by interconnecting information sources of partners.
+Please **[visit the website](http://k-link.technology)** for more information!
 
-## Getting started
+If you find any issues with this application, please report them at the [issue tracker](./issues). Contributions are both encouraged and appreciated. If you like to contribute please check the website for more information.
```

Comment:
I would move this below, before the contribution text, as brake the reading flow